Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
> too risky

The risk being what ? Having less code coverage reported ? Not a big risk IMO

But I think we should not use `@covers` for every method, only for public methods that call other public methods.